### ISSUE TITLE
Enhancement idea: replace the screenshots in the "Tutorial for Cantus contributors" so they match the new look

### ISSUE THREAD

**Issue by JoyfulGen** [DDMAL/CantusDB#983]
The screenshots that support the explanations of each step of the "Tutorial for Cantus contributors" page (https://cantusdatabase.org/tutorial/#Step%204:%20Add%20new%20chant) are taken from OldCantus, so they don't quite match what Cantus looks like now. Everything is almost the same, so this is not an urgent problem at all, but I think it'll eventually be a smoother look if the tutorial matches the site exactly!

**Comment by jacobdgm** [comment]
This sounds like a task for Debra, or perhaps @annamorphism or @JoyfulGen (i.e. someone who's not a developer). Not sure whether it's worth suggesting this to Debra, as it sounds like she already has a lot on her plate.

**Comment by JoyfulGen** [comment]
@jacobdgm wait, do I have that power? If so, and if it suits everyone, I'd be very happy to make these edits.

**Comment by annamorphism** [comment]
I think this sounds like a great task for @JoyfulGen but I don't know if she has the power--for that matter I don't know that _I_ have the power to change the images referenced in the flatpages, which seem to live in /static/some-image.jpg.

**Comment by jacobdgm** [comment]
@JoyfulGen - can you send me or @lucasmarchd01 updated images (or just post them here)? Thinking about this a bit more closely, a developer will have to upload any updated images to our static resources in order that a "staff" like Gen be able to link to them on the flatpages.

**Comment by JoyfulGen** [comment]
Sounds good to me! Pictures coming up. 

@jacobdgm @annamorphism, just to make absolutely sure that I won't do something dumb again: if I create a source to get the screenshots I need, will I be able to delete that source afterwards? I'm just thinking of copying exactly the info that's in the current tutorial screenshots, which means I would be creating a manuscript called "Prague, University library XI C 3." Is that ok?

**Comment by jacobdgm** [comment]
> if I create a source to get the screenshots I need, will I be able to delete that source afterwards?

Yes, if you have access to the admin area, you can delete sources.

(sorry about the delay in getting back to you, I'm only working a few days a week for the lab this summer)
